util/libcrypto.num

```diff
@@ -4382,3 +4382,5 @@ ASN1_TIME_compare                       4325	1_1_1	EXIST::FUNCTION:
 EVP_PKEY_CTX_ctrl_uint64                4326	1_1_1	EXIST::FUNCTION:
 EVP_DigestFinalXOF                      4327	1_1_1	EXIST::FUNCTION:
 ERR_clear_last_mark                     4328	1_1_1	EXIST::FUNCTION:
+_strlen31                               4329	1_1_1	NOEXIST::FUNCTION:
+OPENSSL_strlen31                        4330	1_1_1	EXIST::FUNCTION:
```

Comment:
This is where life gets difficult...  if you only implement it for WIN64, you must also make sure it's only declared for WIN64 in the appropriate header, and for `mkdef.pl` to recognise that.  Thing is, `_WIN64` isn't one of them without modifications.  Quite frankly, it's probably better if you go with the original inline `_strlen31`, then you don't have to mess with this.

Reply:
As discussed in comment, strlen31 should either be omitted, or all 64-bit platforms should use it. But internally. In other words even if we choose latter, i.e. making all 64-bit platforms using it, I'd vote against exporting it.

Reply:
since this PR is closed, do you think we should move forward on doing it for all64bit?  I'm opposed, it silently truncates things.